docs/examples/plot_1D_basis_function.py

```diff
@@ -81,3 +83,87 @@
 plt.plot(samples, eval_basis)
 plt.show()
 
+# %%
+# ### The Fourier Basis
+# Another type of basis available is the Fourier Basis. Fourier basis are ideal to capture periodic and
+# quasi-periodic patterns. Such oscillatory, rhythmic behavior is a common signature of many neural signals.
+# Additionally, the Fourier basis has the advantage of being orthogonal, which simplifies the estimation and
+# interpretation of the model parameters, each of which will represent the relative contribution of a specific
+# oscillation frequency to the overall signal.
+
+
+# A Fourier basis can be instantiated with the usual syntax.
+# The user can pass the desired frequencies for the basis or
+# the frequencies will be set to np.arange(n_basis_funcs//2).
```

Comment:
I don't think description is correct -- there's only one option for initialization, setting `n_freqs`, whereas this makes it sounds like there's two. 

And I think the meaning of `n_freqs` needs to be explained. When `n_freqs=4`, we get cosines and sines from 0 to 3 periods. I think maybe the name of the initialization argument should be changed? To something like `max_freq` (which is the current `n_freq-1`), and then describe the basis set as consisting of cosines and negatives sines from 0 to 3 periods? I would use the term "quadrature pair" here, but my guess is that's not any more transparent to most people...

Reply:
I'll give it a try. Let me know if the description improves!

Reply:
the description was of a earlier version in which I allowed to pass a frequency vector but I changed it to this more restrictive parametrization

Reply:
yup, this looks good now!